templates/source/sourceTemplate.md

```diff
@@ -0,0 +1,35 @@
+# Event Source for **\_\_\_\_**
+
+This event target receives .......
+
+## Prerequisites
+
+- unordered list out prerequisites
+- unordered list out prerequisites
+- unordered list out prerequisites
+
+##### This should be about a prerequisites
+
+1. number list of each step for the prerequisite
+2. number list of each step for the prerequisite
+3. number list of each step for the prerequisite
+
+Or if there are no steps, you can add details about each one.
+
+##### This should be about a prerequisites
+
+1. number list of each step for the prerequisite
+2. number list of each step for the prerequisite
+3. number list of each step for the prerequisite
+
+Or if there are no steps, you can add details about each one.
+
+## Deploying an Instance of the Source
+
+Open the Bridge creation screen and add a source of type **\_\_**
+
+Continue adding descriptions and steps needed.
+
+## Event Types
+
+Add the event type info here.
```

Comment:
```suggestion
Add the event type info here. If schemas are available, add link to those schemas for each type.
```
I think it helps people to know what the events flowing through the bridges look like when planning on connecting sources with targets, and target responses with other targets.